### DISCUSSION TITLE
What's the expected level of service here?

### DISCUSSION THREAD

**Discussion by riverar** [microsoft/WSA#55] [General]
Is the team promising to triage issues in this repository regularly? As soon as they come in? What can folks expect in terms of service level here? (Might be worth noting in the README.)

**Comment by hamza-usmani** [comment]
Hi Rafael! Yes, we will be triaging actionable/relevant issues and aim to have 0 open and untriaged issues by the end of every week. Hope this provides some clarity here.

**Reply by riverar** [reply]
@johnkellyjr You should file a new issue rather than re-use this thread so you get maximum visibility. (UDP datagrams are indeed problematic/blocked, btw.)

**Reply by johnkellyjr** [reply]
Thanks, Riverar. I will delete this.
